### ISSUE TITLE
[Bug] Color scheme toggle is invisible

### ISSUE THREAD

**Issue by ChronoJon** [quarto-dev/quarto-cli#309]
The light/dark mode toggle appears invisible after rendering. This was tested in version 0.3.76 on Windows 10 (Build 19044.1466) on Chrome and FireFox.

`minimal.qmd`
```markdown
---
title: Minimal Button Example
format:
  html:
    theme:
      dark: darkly
      light: flatly
---

# Test Header 1

## Test Subheading

```
  
Calling `quarto render .\minimal.qmd` renders the page like this:
```html
<a id="quarto-color-scheme-toggle" class="top-right" href="">
    <i class="bi"></i>
</a>
```

The button is functional if you make a small change like this:
```html
<a id="quarto-color-scheme-toggle" class="top-right" href="">
    <i class="bi bi-heart"></i>
</a>
```
The JS works. My guess would be, that it's a problem with the scss, where I'm of no help.

**Comment by dragonstyle** [comment]
Thanks for letting us know - I've reproduced this and should have a fix out shortly!

**Comment by dragonstyle** [comment]
Fixed here: fafede6dc3b2261783c8d077705b8d85335c5caf

We're building an installer that includes the fix now, it should be available shortly at:

https://github.com/quarto-dev/quarto-cli/releases/tag/v0.3.91

**Comment by ChronoJon** [comment]
Thanks for the fix. Really love quarto. It's everything, I ever wanted nbconvert to be and more.
